core/src/main/java/com/linecorp/armeria/common/util/EventLoopGroups.java

```diff
@@ -93,5 +97,29 @@ public static EventLoopGroup newEventLoopGroup(int numThreads, ThreadFactory thr
         return type.newEventLoopGroup(numThreads, unused -> threadFactory);
     }
 
+    /**
+     * Returns the {@link ServerChannel} class that is available for this {@code eventLoopGroup}, for use in
+     * configuring a custom {@link Bootstrap}.
+     */
+    public static Class<? extends ServerChannel> serverChannelClass(EventLoopGroup eventLoopGroup) {
```

Comment:
Just realized that this has to be `serverChannelType`. Would you mind sending a follow-up pull request, @anuraaga ?